2_horseshoe_bat/api_definitions/rest/metadata_repository.yaml

```diff
@@ -0,0 +1,2676 @@
+          description: The Data Access Policy that applies to this Dataset.
+          title: Has Data Access Policy
+        has_experiment:
+          anyOf:
+          - items:
+              type: string
+            type: array
+          - items:
+              $ref: '#/components/schemas/Experiment'
+            type: array
+          description: One or more Experiment entities that are referenced by             this
+            Dataset.
+          title: Has Experiment
+        has_file:
+          anyOf:
+          - items:
+              type: string
+            type: array
+          - items:
+              $ref: '#/components/schemas/File'
+            type: array
+          description: One or more File entities that collectively are part of             this
+            Dataset.
+          title: Has File
+        has_publication:
+          anyOf:
+          - items:
+              type: string
+            type: array
+          - items:
+              $ref: '#/components/schemas/Publication'
+            type: array
+          description: One or more Publication entities associated with this Dataset.
+          title: Has Publication
+        has_sample:
+          anyOf:
+          - items:
+              type: string
+            type: array
+          - items:
+              $ref: '#/components/schemas/Sample'
+            type: array
+          description: One or more Sample entities that are referenced by this Dataset.
+          title: Has Sample
+        has_study:
+          anyOf:
+          - items:
+              type: string
+            type: array
+          - items:
+              $ref: '#/components/schemas/Study'
+            type: array
+          description: One or more Study entities that are referenced by this Dataset.
+          title: Has Study
+        id:
+          description: An identifier that uniquely represents an entity.
+          title: Id
+          type: string
+        status:
+          allOf:
+          - $ref: '#/components/schemas/ReleaseStatusEnum'
+          description: The status of an entity.
+        title:
+          description: The title that describes an entity.
+          title: Title
+          type: string
+        type:
+          description: The type of Dataset.
+          title: Type
+          type: string
+        update_date:
+          description: Timestamp (in ISO 8601 format) when the entity was updated.
+          title: Update Date
+          type: string
+        xref:
+          description: Alternate identifiers for an entity.
+          items:
+            type: string
+          title: Xref
+          type: array
+      title: Dataset
+      type: object
+    DatasetStatusPatch:
+      properties:
+        id:
+          description: A unique GHGA identifier assigned to an entity for the sole
+            purpose             of referring to that entity in a global scope.
+          title: Id
+          type: string
```

Comment:
Is there a use case for updating the ID? Should we allow that at all?

Reply:
Good point. 👍